File: 028-php模板引擎原理/template/controller/member.php
<?php
    include '../core/Template.php';
    $tpl = new Template(['php_turn'=>true,'debug'=>true]);
    $tpl->assign('data','hello world');
    $tpl->assign('person','cateCat');
    $tpl->assign('pai','3.14');
    $tpl->assign('b',[1,2,3,4,"hahattt",6]);
    $tpl->show('member');

File: 028-php模板引擎原理/template/template/member.m
<html>
{! like.js}
{$data},{$person}
<ul>
<?php
echo $pai*2;?>
{if $data == 'abc'}
我是abc
{elseif $data == 'def'}
我是def
{else}
我就是我,{$data}
{/if}
</html>